backend/config/routes.yaml

```diff
@@ -20,3 +20,9 @@ updatePlayer:
     path: /player/{id}
     controller: App\Controller\PlayerController::updatePlayer
     methods: PATCH
+
```

Comment:
i have created the new routes.yaml file with the proper configuration, it should be merged anytime soon